e2e/src/deploy/elementWeb/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM node:20-bullseye AS builder
+FROM node:20-bookworm AS builder
 
 ARG ELEMENT_VERSION="master"
 
 WORKDIR /src
 
 # Clone the release tag from element
-RUN git clone --depth 1 --branch $ELEMENT_VERSION https://github.com/element-hq/element-web.git /src
+# Cannot be shallow so we can determine a release version later
+RUN git clone --branch $ELEMENT_VERSION https://github.com/element-hq/element-web.git /src
```

Comment:
We do know the Element version. That is what the crazy build script does I think. Also we have an arg here that is named „ELEMENT_VERSION“. So, why use the build script at all, that only seems to run `yarn` + some version detection?

Reply:
Wait for Element to update the script, that it works with a shallow clone

Reply:
I would like to stick to Element's upstream build script instead of introducing additional maintenance burden to us to keep our copy updated.